cnv/cnv_release_notes/cnv-release-notes.adoc

```diff
@@ -26,6 +26,21 @@ include::modules/technology-preview.adoc[leveloffset=+2]
 
 // CNV-4134 virtctl upload
 
+=== Web console improvements
+
+* You can now view the *Paused* status of a virtual machine in the web console. The web console displays this status on the *Virtual Machines* dashboard and on the *Virtual Machine Details* page. +
+
+[NOTE]
+====
+You can also view the *Paused* status of a virtual machine by using the `oc` CLI:
+----
+$ oc get vmi testvm -o=jsonpath='{.status.conditions[?(@.type=="Paused")].message}'
+----
+====
```

Comment:
@kbidarkar do you think this is ok? It doesn't really belong under "web console improvements" but I want to keep the relevant info together. cc @aburdenthehand if you want to weigh in :) thanks!

Reply:
@ousleyp: Yes looks good now.